jax/experimental/global_device_array.py

```diff
@@ -189,86 +188,85 @@ class GlobalDeviceArray:
   devices, and the data is visible to the current process. Global shards are
   those across all devices (including local devices), and the data isn’t visible
   if the shard is on a non-local device with respect to the current process.
-  Please see the `Shard` class to see what information is stored inside that
+  Please see the ``Shard`` class to see what information is stored inside that
   data structure.
 
   Note: to make pjit output GlobalDeviceArrays, set the environment variable
-  `JAX_PARALLEL_FUNCTIONS_OUTPUT_GDA=true` or add the following to your code:
-  `jax.config.update('jax_parallel_functions_output_gda', True)`
+  ``JAX_PARALLEL_FUNCTIONS_OUTPUT_GDA=true`` or add the following to your code:
+  ``jax.config.update('jax_parallel_functions_output_gda', True)``
+
+  Args:
+    global_shape: The global shape of the array.
+    global_mesh: The global mesh representing devices across multiple
+      processes.
+    mesh_axes: A sequence with length less than or equal to the rank of the
+      global array (i.e. the length of the global shape). Each element can be:
+
+      * An axis name of ``global_mesh``, indicating that the corresponding
```

Comment:
This isn't rendering as a list in the docs. I think this should be possible, can you look up how to do it and fix?

Reply:
Its rendering on my local machine with bullets. This is weird.